src/main/java/com/wavefront/AbstractTraceLoader.java

```diff
@@ -28,7 +30,9 @@
   protected static final Logger LOGGER = Logger.getLogger("traceloader");
   protected final GeneratorConfig generatorConfig = new GeneratorConfig();
   protected ApplicationConfig applicationConfig;
-
+  public static MemoryMXBean memoryMXBean = ManagementFactory.getMemoryMXBean();
```

Comment:
This member can be final.
I don't think it should be public. You can wrap its functionality into your own methods

Reply:
This member can be final, but I maked it public, because of i use this member in another class TraceGenerator in updateHeapMemory() method